Update Swift docs in included_software.md (#495)
netlify/build-image: included_software.md

  * 21 (default)
* Elixir
  * 1.7 (default)
* Swift - `SWIFT_VERSION`, `.swift-version`
  * Not installed by default.

>   * Supports any version that `swiftenv` can install newer than `4.x`. Versions `4.x` and below will not work due to incompatible shared libraries.
>   * 5.2 will be installed if `Package.swift` is present and no version is specified.

Reviewer: [sand] some wording suggestions here:
```suggestion
  * Supports any version that `swiftenv` can install later than `4.x`. Versions `4.x` and earlier will not work due to incompatible shared libraries.
  * 5.2 is installed if `Package.swift` is present and no version is specified with `SWIFT_VERSION` or `.swift-version`.
```
